cap_anndata/cap_anndata.py

```diff
@@ -136,7 +145,10 @@ def overwrite(self, fields: List[str] = None) -> None:
 
                 for col in entity.columns:
                     self._write_elem_lzf(f"{key}/{col}", entity[col].values)
-                self._file[key].attrs['column-order'] = entity.column_order
+
+                column_order = entity.column_order
+                if column_order.size > 1: # Refs https://github.com/cellannotation/cap-anndata/issues/6
```

Comment:
It will not work:, we need to rewrite it in any case. but in case of 0 columns looks like we need to handle specian case. This approach ruins the file, see updated unit test

Reply:
looks like fixed now